packages/mux-audio/tsconfig.json

```diff
@@ -2,11 +2,13 @@
   "compilerOptions": {
     "preserveWatchOutput": true,
     "rootDir": "src",
-    "outDir": "dist",
+    "outDir": "./dist/types",
+    "declaration": true,
+    "emitDeclarationOnly": true,
     "incremental": true,
     "target": "ES2019",
     "module": "es6",
-    "declaration": true,
+    "jsx": "react",
```

Comment:
nitpick(non-blocking): Unneeded config prop.

Reply:
I was thinking about it, the main reason it's here is to keep the tsconfigs across all the repos as close to each other as possible. However, I can def remove it if you think it's not worth having for the non-react packages.

Reply:
nah I'm cool with keeping it around, esp since tsconfig can inherit.

Reply:
actually, that's something to consider for the future. Have the top level tsconfig with like 90% of the stuff and have each package inherit from that one.